# Luna::NetworkError::address_in_use

```c++
ErrCode address_in_use()
```

The speciifed address is already bound to one existing socket. 

